### DISCUSSION TITLE
General Issue: Explore parallelisation in admiral

### DISCUSSION THREAD

**Discussion by manciniedoardo** [pharmaverse/admiral#2606] [General]
### Background Information

Can we explore how to leverage packages such as {parallel} to make the execution of functions such as `derive_vars_joined()` faster?

### Definition of Done

- [ ] Explore parallelisation options
- [ ] Agree on an option and implement package-wide.

**Comment by bms63** [comment]
I really think this is a Discussion topic more than an issue.  Can you push into Discussions?

@pharmaverse/admiral @pharmaverse/admiral_comm Anyone want to put on their explorer cap!?
